openbb_platform/platform/provider/openbb_provider/abstract/data.py

```diff
@@ -15,7 +20,7 @@ def check_int(v: int) -> int:
         raise TypeError("value must be an int") from exc
 
 
-StrictInt = Annotated[int, BeforeValidator(check_int)]
+ForceInt = Annotated[int, BeforeValidator(check_int)]
```

Comment:
Why do we have to use `ForceInt`?

Reply:
Randomly FMP will have float on expected int fields and give validation error, this keeps the functionality that pydantic < v2 had, which was `int(value)` on int fields
![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/61340027/b4e3e400-eb15-40c5-9b61-c8cab952f076)